CardoExecutor/Common/CardoWrapper.py

```diff
@@ -0,0 +1,25 @@
+_INNER = "_inner"
```

Comment:
correct me if im wrong but this wrapper only used on dataframes.
if so i think it will be better to call it CardoDataframeWrapper or DataframeWrapper

Reply:
nothing couples it to dataframes (even now it also supports RDD :))
you're right there are not many other objects we might want to wrap - but a SparkSession might be one of them.

Reply:
in that case i would change the `_get_attribute` signature to something like:
def _get_attribute(cardo_object, item, item_type, attribute)
because it will be funny to send a SparkSession to a variable called `df` or `df_type`

Reply:
OK